Bottom: f24383b3d69f121873ada96d0d28bfc9507c601f
Top:    bb43dd1ec721f81531d366abcf1b050512414f9c
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:25:11 +0800

Refresh of macvlan-xdp-rx-handler-support

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index c3b0758..e2f448f 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -1202,8 +1202,7 @@ static int macvlan_port_create(struct net_device *dev)
 	INIT_WORK(&port->bc_work, macvlan_process_broadcast);
 
 	err = netdev_rx_handler_register_xdp(dev, macvlan_handle_frame,
-					     macvlan_handle_frame_xdp,
-					     port);
+					     macvlan_handle_xdp, port);
 
 	if (err)
 		kfree(port);
